test/test_files/common/arrayimplicitcasting.test

```diff
@@ -3,6 +3,21 @@
 --
 
 -CASE ArrayImplicitCast
+-STATEMENT call threads=4.5
+---- error
+Binder exception: Invalid cast from DOUBLE to UINT64 during threads.
```

Comment:
I think it is better to make the error message consistent.
I suggest change it as:
Binder exception: Expression 4.5 has data type DOUBLE but expected UINT64. Implicit cast is not supported.